ada/language/ast.py

```diff
@@ -3030,12 +3030,21 @@ def xref_equation():
     xref_entry_point = Property(True)
 
 
+class DiscriminantCondition(Struct):
```

Comment:
I don't much like the term `Condition` here, I think it's pretty vague. What about `DiscriminantValues` with docstring Represent a set of values (as a list of choices) for a discriminant.